### DISCUSSION TITLE
Add ability to use TrueType fonts when integrated in an osu! skin

### DISCUSSION THREAD

**Discussion by heyitsjustpIxel** [ppy/osu#30557] [Ideas]
It would be useful for skinning if the osu!lazer skin editor supported TrueType (TTF) fonts when designing skins. (if the TTF file is integrated inside the skin folder) 

Currently, the editor seems limited to built-in fonts, since theres Venera, Tarus and Inter already integrated.

**Comment by RUvlad1** [comment]
I guess with that it would also be nice if osu! could show the fonts that are currently installed on your system
